README.md

```diff
@@ -248,6 +275,16 @@ This is why Lua, with a single extra operator that was
 easy to implement is what I went with in Hawck.
 </small>
 
+The lua files are deleted on the fly, but you can open a lua shell with your
+script loaded for investigation:
+
+```sh
+cd ~/.local/share/hawck/scripts/ ; \
+hwk2lua example.hwk >! tmp.lua ; \
+lua -l init -l tmp -i ; \
```

Comment:
Remove the trailing ` ; \`s and add leading `$`s so that it fits with the other command line code samples from the README.

Reply:
done, but this defeats copy-paste :D